compiler/rustc_ast_lowering/Cargo.toml

```diff
@@ -9,6 +9,7 @@ doctest = false
 [dependencies]
 # tidy-alphabetical-start
 rustc_ast = { path = "../rustc_ast" }
+rustc_ast_pretty = { path = "../rustc_ast_pretty" }
```

Comment:
Was this already a transitive dep, or did this change reduce build parallelism?

Reply:
It was a dep through rustc_errors (and it also was a direct dep ~10 days ago: https://github.com/rust-lang/rust/commit/b480f0f224b6e92a514a477516dd65d06b68f3cc)